composer.json

```diff
@@ -20,7 +20,7 @@
                 "version": "0.3.0-rc27",
                 "type": "drupal-library",
                 "dist": {
-                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/release-develop/dist.zip",
```

Comment:
Please use the develop version of `danskernesdigitalebibliotek/dpl-react`. 
If the `dpl-react:release-fix/dashboard-array-length-prints` hasn't been merged yet please make sure that it is merged first.
Until now that has been our rule of thumb: Dependencies merged into develop should also point at develop releases.